check50/__main__.py

```diff
@@ -385,9 +432,10 @@ def main():
             # Have lib50 decide which files to include
             included = lib50.files(config.get("files"))[0]
 
-            with open(os.devnull, "w") if args.verbose else nullcontext() as devnull:
-                # Redirect stdout to devnull if some verbosity level is set
-                if args.verbose:
+            # Redirect stdout to devnull if verbose or log level is set
+            should_redirect_devnull = args.verbose or args.log_level
+            with open(os.devnull, "w") if should_redirect_devnull else nullcontext() as devnull:
```

Comment:
Wait, was this even right in the first place? Shouldn't we redirect output to /dev/null if we _aren't_ in verbose mode?

Reply:
Yup this is wrong.